src/http/method.rs

```diff
@@ -26,6 +28,25 @@ pub enum Method {
     TRACE,
 }
 
+impl FromStr for Method {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "GET" => Ok(Self::GET),
+            "POST" => Ok(Self::POST),
+            "PUT" => Ok(Self::PUT),
+            "PATCH" => Ok(Self::PATCH),
+            "DELETE" => Ok(Self::DELETE),
+            "HEAD" => Ok(Self::HEAD),
+            "OPTIONS" => Ok(Self::OPTIONS),
+            "CONNECT" => Ok(Self::CONNECT),
+            "TRACE" => Ok(Self::TRACE),
+            _ => Err("Wrong method name"),
+        }
+    }
+}
+
```

Comment:
I don't think we need this

Reply:
Yeh, It was not needed, Removed in the following commit.